packages/api/lib/reconciliationReport.js

```diff
@@ -113,18 +129,24 @@ function initialReportHeader(recReportParams) {
     createStartTime,
     endTimestamp,
     startTimestamp,
+    granuleIds,
+    granuleId,
+    collectionIds,
     collectionId,
   } = recReportParams;
 
   return {
-    reportType,
-    createStartTime: createStartTime.toISOString(),
+    collectionId,
+    collectionIds,
```

Comment:
I think we only need `collectionId` to match the event input

Reply:
> It looks good to me. I haven't tested it yet. Are you going to add `granuleId` parameter to integration test spec?

I was not going to because it would require another complete run of the reconcilation report as we can't have both collectionId and granuleId on the same request.

Reply:
That is true when your work gets finished, but right now, the normalizeEvent code, for non 'Internal' types moves the input string or array<string> from collectionId to collectionIds. (same for granuleId) (or it should.)

Reply:
I think your PR that handles multiple collectionId and granuleId should stop that behavior and just transform the input string or array into an array on the singular version... eg.
if the input is collectionId = "collectionId" => collectionId = ["collectionId"] (rather than moving it to "collectionIds")

Reply:
I am open to something else.  I tried also just adding the entire recReportParams key, but it had duplicates, It might be best to do that because we will keep adding and removing parameters and want to know exactly how the lambda was called in our report.